website/integrations/services/nextcloud/index.md

```diff
@@ -23,10 +27,11 @@ In case something goes wrong with the configuration, you can use the URL `http:/
 
 ## Authentication
 
-There are 2 ways to setup single sign on (SSO) for Nextcloud:
+There are 3 ways to setup single sign on (SSO) for Nextcloud:
 
 -   [via OIDC Connect (OAuth)](#openid-connect-auth)
 -   [via SAML](#saml-auth)
+-   via LDAP outpost
```

Comment:
Since we are adding a third option here, we need to add a section below for it, like the other two. This new sectin could be quite short, just a sentence explaining the process conceptually, and link off to this doc: https://docs.goauthentik.io/docs/add-secure-apps/outposts/#more-about-outpost-integrations.

Or link to the separate docs for K9s vs Docker...

Reply:
It is not trivial and I do not have a clean configuration to offer, much less a reasonable process. I thought the **data loss** warranted a notice. I am not agreeing to expand the scope of the volunteer work I am doing at this time.

Reply:
No worries, I can do it later, after we merge your addition! Thanks again for adding the important info.